### PR TITLE
MLE-12851 Added Gradle test project to verify MLCP and its dependencies

### PR DESCRIPTION
**Pull request by rjrudin**

### CODE REVIEW

File: src/test/resources/gradle-test-project/build.gradle [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+configurations {
+    mlcp {
+        attributes {
+            attribute(TargetJvmEnvironment.TARGET_JVM_ENVIRONMENT_ATTRIBUTE, objects.named(TargetJvmEnvironment.class, TargetJvmEnvironment.STANDARD_JVM))
+        }
+    }
+}
+
+dependencies {
+    mlcp "com.marklogic:mlcp:12.0"
```
Comment (yunzvanessa) [inline src/test/resources/gradle-test-project/build.gradle:27]: Hi Rob,

Should this be mlcp 11.2?

**Review by yunzvanessa — COMMENTED** [review]
Please see my comments. Thanks!

**Comment by yunzvanessa** [comment]
Also, sorry forgot to mention develop-11 is under control checkin right now for 11.2.0 release. We may want to hold this PR until it opens.

Thanks,
Vanessa

File: src/test/resources/gradle-test-project/build.gradle (hunk 1 above)
Comment (rjrudin) [reply src/test/resources/gradle-test-project/build.gradle:27]: Yes, it should be 11.2. There's likely a way to read it from the pom.xml file, but I think it's going to be changed infrequently enough that it's fine to hard-code it.

**Review by yunzvanessa — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**